src/Type/Row.purs

```diff
@@ -0,0 +1,27 @@
+module Type.Row
+  ( class RowLacks
+  ) where
+
+-- Must not be exported
+foreign import data RowLacking :: Symbol -> Symbol -> Type
+
+class RowLacksF (key :: Symbol)
```

Comment:
That's not quite correct, it doesn't say anything about **all** positions, only that if you add `key :: typ` on the right then you get back `typ` on the left if you uncons the row at `key`. I've updated with comments :)